fdbserver/masterserver.actor.cpp

```diff
@@ -120,6 +120,7 @@ struct MasterData : NonCopyable, ReferenceCounted<MasterData> {
 };
 
 ACTOR Future<Void> getVersion(Reference<MasterData> self, GetCommitVersionRequest req) {
+	// TODO - ljoswiak link or parent?
```

Comment:
I see how this could be a link, as this is a version being generated for a batch. However, since the parent of this span is the span from the request, I think parent may make more sense. Happy to defer to you though if you think differently.

Reply:
Passing as parent, removing TODO